### PR TITLE
Making changes to read tracks 

### PR DESCRIPTION
**Pull request by scandindian**
Currently the App uses Drupal SDK for its data and DrupalClient to read that, stored using model and DAO system.

We fetch data from our samples and use them instead of Drupal SDK. 

First step for issue https://github.com/fossasia/connfa-android/issues/4 . I'll add subsequently for processing sessions, microlocations etc.

### CODE REVIEW

**Comment by scandindian** [comment]
@niranjan94 @the-dagger @Shailesh351 Can you see please?

File: app/src/main/java/com/ls/api/Processor.java [hunk 1]
```diff
@@ -1,8 +1,51 @@
...
+            for(i=0;i<tracks.length();i++)
+            {
+                JSONObject tracksJSONObject = tracks.getJSONObject(i);
+
+                track = new Track();
+                track.setId(tracksJSONObject.getLong("id"));
+                track.setName(tracksJSONObject.getString("name"));
+
+                listTrack.add(track);
+            }
```
Comment (iamareebjamal) [inline app/src/main/java/com/ls/api/Processor.java:43]: Can't we use GSON for this?

Comment (scandindian) [reply app/src/main/java/com/ls/api/Processor.java:43]: @iamareebjamal I was looking not to use gson so I can have processing units for all the parts in one class unlike having serialisation tags in model to have similar structure with open event android app as it doesn't uses gson too. I'll remove that once all the managers are modified.

Comment (iamareebjamal) [reply app/src/main/java/com/ls/api/Processor.java:43]: OK

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**